distributed/tensor_parallelism/README.md

```diff
@@ -12,5 +12,5 @@ https://pytorch.org/docs/stable/distributed.tensor.parallel.html
 
 ```
 pip install -r requirements.txt
-python example.py
+torchrun --nnodes 1 --nproc-per-node 4 tensor_parallel_example.py
```

Comment:
There are other example scripts:
* fsdp_tp_example.py
* llama2_model.py
* sequence_parallel_example.py
* tensor_parallel_example.py

Items above describe 3 cases by the way, but there are 4 scripts - that's not quite clear.

Why you did not update llama2 and fsdp scripts by the way?